### PR TITLE
SEO-171884 ASP.NET Core - Permanent Redirect

### PR DESCRIPTION
**Pull request by Thahira-Banu**
Hi @Aishwarya-Ganesan 

I have made changes in the permanent redirect links by changing the title,  h1 tag,URL

### CODE REVIEW

File: aspnet-core/Overview.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: post
-title: Welcome to Syncfusion Essential ASP.NET Core
+title: Overview for ASP.NET Core Application | Syncfusion
```
Comment (gowthamanthavasiyappan) [inline aspnet-core/Overview.md:3]: Recheck this title.

**Review by gowthamanthavasiyappan — CHANGES_REQUESTED** [review]
Commented

**Review by gowthamanthavasiyappan — CHANGES_REQUESTED** [review]
Commented

File: aspnet-core/Overview.md [hunk 2]
```diff
@@ -21,13 +21,13 @@ Essential JS is a comprehensive collection of over 80 enterprise-grade HTML5 Jav
 
 * After you have integrated the component into your application using one of the sample code snippets as a starting point, it is likely that you would want additional information on specific features and API. The best option would be to search a the specific topic using the search box that is available at the top of the user guide.
 
-* Another valuable resource is the [API reference](https://help.syncfusion.com/cr/aspnet-core) which provides detailed information on the object hierarchy as well as the settings available on every object.
+* Another valuable resource is the [API reference](https://help.syncfusion.com/cr/aspnet-core/Syncfusion.CompoundFile.DocIO.html) which provides detailed information on the object hierarchy as well as the settings available on every object.
```
[the remark this answers is not in the document]
Comment (Thahira-Banu) [reply aspnet-core/Overview.md:24]: Checked, this is fine to change

File: aspnet-core/Overview.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: post
-title: Welcome to Syncfusion Essential ASP.NET Core
+title:  Welcome to Syncfusion Essential ASP.NET Core
```
Comment (gowthamanthavasiyappan) [inline aspnet-core/Overview.md:3]: Remove unwanted space here.

**Review by gowthamanthavasiyappan — CHANGES_REQUESTED** [review]
Commented

File: aspnet-core/Overview.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: post
-title: Welcome to Syncfusion Essential ASP.NET Core
+title: About ASP.NET Core Application | Syncfusion
```
[the remark this answers is not in the document]
Comment (Thahira-Banu) [reply aspnet-core/Overview.md]: Resolved

File: aspnet-core/Overview.md (hunk 3 above)
Comment (Thahira-Banu) [reply aspnet-core/Overview.md:3]: Resolved

**Review by gowthamanthavasiyappan — APPROVED** [review]
Approved

**Review by Girija-Rajendran — APPROVED** [review]

**Review by Girija-Rajendran — APPROVED** [review]

**Review by Girija-Rajendran — APPROVED** [review]

**Pull request merged**